libyul/Utilities.h

```diff
@@ -59,4 +59,11 @@ struct Less<T*>
 template<> bool Less<Literal>::operator()(Literal const& _lhs, Literal const& _rhs) const;
 extern template struct Less<Literal>;
 
+struct SwitchCaseSorter
```

Comment:
Being pedantic this is more a compare than a sorter :p

Reply:
Well, true :-). Unfortunately it's not really a ``Less<Case>``, since for that the bodies would have to be sorted as well... but I'll rename it.

Reply:
Renamed it to be entirely explicit.